doc/release-notes/8639-computational-workflow.md

```diff
@@ -0,0 +1,6 @@
+## Adding Computational Workflow Metadata
+The new Computational Workflow metadata block will allow depositors to effectively tag datasets as computational workflows.
+
+To add the new metadata block, follow the instructions in the user guide: <https://guides.dataverse.org/en/latest/admin/metadatacustomization.html>
+
+The location of the new metadata block tsv file is: `dataverse/scripts/api/data/metadatablocks/computational_workflow.tsv`
```

Comment:
I am not sure it should be. Is this something you can make a decision on @jggautier?

Or @scolapasta / @pdurbin , should this feature be labeled as experimental?

Reply:
@poikilotherm, the effect of labelling this experimental would be to let installations know that it's likely that it will change (more likely than other changes in the update) as more work is done to support the publication of workflows and software. Is that the reasoning?

Since this is considered an MVP, I agree, and the group that worked on this metadatablock share the same concerns you raise in your comments in this PR (and in your talks during the Dataverse Community Meeting). I would confirm with Mahmood that this be labelled as experimental. Does anyone know Mahmood's GitHub username?

Reply:
It seems to be @mmshad.

As @scolapasta and @jggautier know, we just talked about this at standup. That is, should this feature be enabled for all new installations or not? We're going to talk about this a bit more during our sprint planning meeting in a few hours.

As Gustavo pointed out, for embargo (which isn't on by default) we turned it on for the demo server to play around with it. (I'm not even sure if embargo is enabled in Harvard Dataverse or not.) So we could try this... enable the workflow block on the demo server first.

If we do flag the workflow block as experimental and not enabled by default we should probably remove it from the appendix of the User Guide or at least have a note saying that your admin need to manually enable it.

@mmshad if you happen to know any Harvard Dataverse users that would like to try out this workflow block, please let us know.